extensions/sql-bindings/src/common/azureFunctionsUtils.ts

```diff
@@ -335,15 +335,15 @@ export async function promptForObjectName(bindingType: BindingType, connectionIn
 		// get connectionURI and selectedDatabase to be used for listing tables query request
 		connectionURI = await getConnectionURI(connectionInfo);
 		if (!connectionURI) {
-			// User cancelled or mssql connection error
+			// mssql connection error
 			// we will then prompt user to choose a connection profile again
```

Comment:
I'd suggest removing these comments saying what is going to happen since that's controlled by the calling code, not this. So if what we do there ever changes or we use this elsewhere this could easily be incorrect.